diff --git a/score/mw/com/impl/BUILD b/score/mw/com/impl/BUILD
index 3d3363b7..63251a29 100644
--- a/score/mw/com/impl/BUILD
+++ b/score/mw/com/impl/BUILD
@@ -985,6 +985,9 @@ cc_test(
 cc_gtest_unit_test(
     name = "service_discovery_test",
     srcs = ["service_discovery_test.cpp"],
+    visibility = [
+        "//visibility:public",
+    ],
     deps = [
         ":impl",
         ":runtime_mock",
@@ -999,6 +1002,9 @@ cc_gtest_unit_test(
 cc_gtest_unit_test(
     name = "generic_proxy_test",
     srcs = ["generic_proxy_test.cpp"],
+    visibility = [
+        "//visibility:public",
+    ],
     deps = [
         ":impl",
         ":runtime_mock",
@@ -1014,6 +1020,9 @@ cc_gtest_unit_test(
 cc_gtest_unit_test(
     name = "sample_reference_tracker_test",
     srcs = ["sample_reference_tracker_test.cpp"],
+    visibility = [
+        "//visibility:public",
+    ],
     deps = [
         ":sample_reference_tracker",
         "@score_baselibs//score/language/futurecpp",
@@ -1023,6 +1032,9 @@ cc_gtest_unit_test(
 cc_gtest_unit_test(
     name = "flag_owner_test",
     srcs = ["flag_owner_test.cpp"],
+    visibility = [
+        "//visibility:public",
+    ],
     deps = [
         ":flag_owner",
     ],
@@ -1031,6 +1043,9 @@ cc_gtest_unit_test(
 cc_gtest_unit_test(
     name = "service_element_type_test",
     srcs = ["service_element_type_test.cpp"],
+    visibility = [
+        "//visibility:public",
+    ],
     deps = [
         ":service_element_type",
         "@score_baselibs//score/mw/log",
@@ -1045,6 +1060,9 @@ cc_unit_test(
         # These tests catch exceptions instead of using gtest EXPECT_DEATH so we disable aborts_upon_exception.
         "-aborts_upon_exception",
     ],
+    visibility = [
+        "//visibility:public",
+    ],
     deps = [
         ":runtime",
         ":runtime_mock",
@@ -1066,6 +1084,9 @@ cc_unit_test(
 cc_gtest_unit_test(
     name = "find_service_handle_test",
     srcs = ["find_service_handle_test.cpp"],
+    visibility = [
+        "//visibility:public",
+    ],
     deps = [
         ":find_service_handle",
         "@score_baselibs//score/mw/log",
@@ -1076,6 +1097,9 @@ cc_gtest_unit_test(
 cc_gtest_unit_test(
     name = "proxy_field_test",
     srcs = ["proxy_field_test.cpp"],
+    visibility = [
+        "//visibility:public",
+    ],
     deps = [
         ":impl",
         ":runtime_mock",
@@ -1088,6 +1112,9 @@ cc_gtest_unit_test(
 cc_gtest_unit_test(
     name = "proxy_event_test",
     srcs = ["proxy_event_test.cpp"],
+    visibility = [
+        "//visibility:public",
+    ],
     deps = [
         ":impl",
         ":runtime_mock",
@@ -1105,6 +1132,9 @@ cc_gtest_unit_test(
         # These tests catch exceptions instead of using gtest EXPECT_DEATH so we disable aborts_upon_exception.
         "-aborts_upon_exception",
     ],
+    visibility = [
+        "//visibility:public",
+    ],
     deps = [
         ":error",
         ":impl",
@@ -1122,6 +1152,9 @@ cc_gtest_unit_test(
 cc_gtest_unit_test(
     name = "generic_skeleton_test",
     srcs = ["generic_skeleton_test.cpp"],
+    visibility = [
+        "//visibility:public",
+    ],
     deps = [
         ":generic_skeleton",
         ":runtime_mock",
@@ -1142,6 +1175,9 @@ cc_gtest_unit_test(
         "generic_skeleton_event_test.cpp",
         "service_discovery_client_mock.h",
     ],
+    visibility = [
+        "//visibility:public",
+    ],
     deps = [
         ":generic_skeleton",
         ":generic_skeleton_event",
@@ -1161,6 +1197,9 @@ cc_gtest_unit_test(
         "traits_test.cpp",
     ],
     features = COMPILER_WARNING_FEATURES,
+    visibility = [
+        "//visibility:public",
+    ],
     deps = [
         ":runtime_mock",
         ":service_discovery_mock",
@@ -1182,6 +1221,9 @@ cc_gtest_unit_test(
         "skeleton_base_test.cpp",
     ],
     features = COMPILER_WARNING_FEATURES,
+    visibility = [
+        "//visibility:public",
+    ],
     deps = [
         ":service_discovery_mock",
         "//score/mw/com/impl/bindings/mock_binding",
@@ -1201,6 +1243,9 @@ cc_gtest_unit_test(
         # These tests catch exceptions instead of using gtest EXPECT_DEATH so we disable aborts_upon_exception.
         "-aborts_upon_exception",
     ],
+    visibility = [
+        "//visibility:public",
+    ],
     deps = [
         ":error",
         "@score_baselibs//score/language/futurecpp",
@@ -1217,6 +1262,9 @@ cc_gtest_unit_test(
         # These tests catch exceptions instead of using gtest EXPECT_DEATH so we disable aborts_upon_exception.
         "-aborts_upon_exception",
     ],
+    visibility = [
+        "//visibility:public",
+    ],
     deps = [
         ":error",
         ":error_serializer",
